src/js/handler/daygrid/moveGuide.js

```diff
@@ -0,0 +1,279 @@
+DayGridMoveGuide.prototype._highlightScheduleBlocks = function(model, parent) {
+    var elements = domutil.find(config.classname('.weekday-schedule'), parent, true);
+
+    util.forEach(elements, function(el) {
+        el.style.margin = '0';
+
+        if (!model.isFocused) {
+            el.style.backgroundColor = el.style.color;
+            el.style.borderLeftColor = el.style.color;
+            el.style.color = '#ffffff';
+        }
+    });
+};
+
+/**
+ * Refresh guide element.
+ * @param {number} leftPercent - left percent of guide element.
+ * @param {number} widthPercent - width percent of guide element.
+ * @param {boolean} isExceededLeft - schedule start is faster then render start date?
+ * @param {boolean} isExceededRight - schedule end is later then render end date?
+ */
+DayGridMoveGuide.prototype.refreshGuideElement = function(leftPercent, widthPercent, isExceededLeft, isExceededRight) {
+    var guideElement = this.guideElement;
+
+    reqAnimFrame.requestAnimFrame(function() {
+        guideElement.style.left = leftPercent + '%';
+        guideElement.style.width = widthPercent + '%';
+
+        if (isExceededLeft) {
+            domutil.addClass(guideElement, config.classname('weekday-exceed-left'));
+        } else {
+            domutil.removeClass(guideElement, config.classname('weekday-exceed-left'));
+        }
+
+        if (isExceededRight) {
+            domutil.addClass(guideElement, config.classname('weekday-exceed-right'));
+        } else {
+            domutil.removeClass(guideElement, config.classname('weekday-exceed-right'));
+        }
+    });
+};
+
+/**
+ * Get schedule block information from schedule data.
+ *
+ * For example, there is single schedule has 10 length. but render range in view is 5 then
+ * rendered block must be cut out to render properly. in this case, this method return
+ * how many block are cut before rendering.
+ *
+ * 이벤트 데이터에서 이벤트 블록 엘리먼트 렌더링에 대한 필요 정보를 추출한다.
+ *
+ * ex) 렌더링 된 블록의 길이는 5지만 실제 이 이벤트는 10의 길이를 가지고 있을 때
+ * 좌 우로 몇 만큼 잘려있는지에 관한 정보를 반환함.
+ * @param {object} dragStartEventData - schedule data from DayGrid.Move handler.
+ * @returns {function} function that return schedule block information.
+ */
+DayGridMoveGuide.prototype._getScheduleBlockDataFunc = function(dragStartEventData) {
+    var model = dragStartEventData.model,
+        datesInRange = dragStartEventData.datesInRange,
+        range = dragStartEventData.range,
+        baseWidthPercent = (100 / datesInRange),
+        originScheduleStarts = datetime.start(model.start),
+        originScheduleEnds = datetime.end(model.end),
+        renderStartDate = datetime.start(range[0]),
+        renderEndDate = datetime.end(range[range.length - 1]),
+        fromLeft = (new TZDate(originScheduleStarts.getTime() -
+            renderStartDate.getTime())) / datetime.MILLISECONDS_PER_DAY | 0,
+        fromRight = (new TZDate(originScheduleEnds.getTime() -
+            renderEndDate.getTime())) / datetime.MILLISECONDS_PER_DAY | 0;
```

Comment:
`I` is bitwise operators. - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_Operators

Reply:
Thank for your review.